source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -80,6 +80,56 @@ Content-Type: application/json
   ]
 }
 ```
+
+## Validating the JWT
+The public keys you must use to validate this JWT are published by OneLogin as [Decentralized Identifiers (DIDs)](https://www.w3.org/TR/did-core/) and can be downloaded from one of the following endpoints:
+
+| Environment | Endpoint                                                         |
+| ----------- | ---------------------------------------------------------------- |
+| Integration | https://identity.integration.account.gov.uk/.well-known/did.json |
+| Production  | https://identity.account.gov.uk/.well-known/did.json             |
+
+You should read this endpoint when validating a document, however to reduce latency we recommend that you incorporate a caching strategy in your service. The endpoint incorporates cache-control headers to facilitate this.
+
+The following is an example of a DID Document which may be downloaded from the OneLogin service. The service will always include one or more `assertionMethod` entries.
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://www.w3.org/ns/security/jwk/v1"
+  ],
+  "id": "did:web:identity.account.gov.uk",
+  "assertionMethod": [
+    {
+      "id": "cfeebabeeac2d9749993523f143fbc3f8c83411853f2996323a2efbd7acda754",
+      "type": "JsonWebKey",
+      "controller": "did:web:staging.identity.account.gov.uk",
+      "publicKeyJwk": {
+        "kty": "EC",
+        "crv": "P-256",
+        "x": "1_WJai9R0PRJrw12kRRq_dk1Kh5eSll7QYC2JTrY0Hg",
+        "y": "Wm5YhRJSsXVmAFGCEUgNBJyvR-oAQo2xvYA9DMVMYdk"
+      }
+    }
+  ]
+}
+```
+
+When validating a JWT the JWT header will include a `kid` (Key ID) in the format `{scheme}:{DID-method}:{DID-method-specific identifier}#{unique key identifier}`. The `kid` can be used to determine which public key was used to the sign the JWT:
```

Comment:
Might be worth link out to the DID `web` method spec to describe how to resolve the kid to a DID document ... specifically [this section](https://w3c-ccg.github.io/did-method-web/#read-resolve).

Reply:
Basically, giving us the option to move where we locate the DID document, though we're unlikely to do so any time soon.

Reply:
Updated the paragraph with this information.